utils/src/main/kotlin/DiskCache.kt

```diff
@@ -45,7 +45,7 @@ class DiskCache(
         /**
          * Duration in seconds that cache entries are valid.
```

Comment:
Commit message: Let's provide a short rationale *why* we rename, i.e. say "DiskCache: Use a more speaking name for a variable" (or property).

Reply:
Done.